group_vars/all/all.yaml

```diff
@@ -5,19 +5,19 @@ backup:
   name: just-ask-backup
 api:
   name: just-ask-api
-  image_name: wazzacky/just-ask-api
-  image_tag: '1.1.106'
-  image_registry: 'docker.io'
+  image_name: plat-gen-docker-local/just-ask
+  image_tag: 'latest'
+  image_registry: 'artifacts.developer.gov.bc.ca'
   port: "3001"
   cpu_limit: 200m
   cpu_request: 100m
   memory_limit: 100Mi
   memory_request: 250Mi
 web:
   name: just-ask-web
-  image_name: wazzacky/just-ask-web
-  image_tag: '1.1.104'
-  image_registry: 'docker.io'
+  image_name: plat-gen-docker-local/just-ask-api
```

Comment:
`just-ask-web` uses api image? also it's better to use specific image tag instead of `latest`.